ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/BlockMiner.java

```diff
@@ -123,28 +129,41 @@ protected boolean mineBlock() throws InterruptedException {
         "Block created, importing to local chain, block includes {} transactions",
         block.getBody().getTransactions().size());
 
-    final BlockImporter<C> importer =
-        protocolSchedule.getByBlockNumber(block.getHeader().getNumber()).getBlockImporter();
-    final boolean blockImported =
-        importer.importBlock(protocolContext, block, HeaderValidationMode.FULL);
-    if (blockImported) {
-      notifyNewBlockListeners(block);
-      final double taskTimeInSec = stopwatch.elapsed(TimeUnit.MILLISECONDS) / 1000.0;
+    if (gpuMining) {
       LOG.info(
-          String.format(
-              "Produced and imported block #%,d / %d tx / %d om / %,d (%01.1f%%) gas / (%s) in %01.3fs",
-              block.getHeader().getNumber(),
-              block.getBody().getTransactions().size(),
-              block.getBody().getOmmers().size(),
-              block.getHeader().getGasUsed(),
-              (block.getHeader().getGasUsed() * 100.0) / block.getHeader().getGasLimit(),
-              block.getHash(),
-              taskTimeInSec));
+            String.format(
```

Comment:
This is formatted incorrectly.  `./gradlew spotlessApply` will fix all formatting errors.